generators/generator-constants.js

```diff
@@ -25,7 +25,8 @@ export const BLUEPRINT_API_VERSION = 'jhipster-8';
 export const JHIPSTER_DEPENDENCIES_VERSION = '8.6.1-SNAPSHOT';
 // Version of Java
 export const JAVA_VERSION = '17';
-export const JAVA_COMPATIBLE_VERSIONS = ['17', '18', '19', '20', '21', '22'];
+// Supported Java versions, https://www.oracle.com/br/java/technologies/java-se-support-roadmap.html
```

Comment:
@mshima É melhor usar o link inglês ;)
```suggestion
// Supported Java versions, https://www.oracle.com/java/technologies/java-se-support-roadmap.html
```